docs/installation.rst

```diff
@@ -237,15 +237,8 @@ end of your urlconf. You can also put it in */wiki* by putting
     and media files from ``STATIC_ROOT`` and ``MEDIA_ROOT`` served by the
     development server. ``STATIC_ROOT`` is automatically served, but you have
     to add ``MEDIA_ROOT`` manually::
-    
-        if settings.DEBUG:
-            urlpatterns += staticfiles_urlpatterns()
-            urlpatterns += patterns('',
-                                    url(r'^media/(?P<path>.*)$',
-                                        'django.views.static.serve',
-                                        {'document_root': settings.MEDIA_ROOT,
-                                         }),
-                                    )
+    urls.py
+        urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
The block below `manually::` has to be indented, but I can see your point - the pattern is outdated for newer versions of Django, and since support is being cut to Django 1.8 (or even more restrictive, that's pending a decision), we can clean up these instructions...